src/components/NavDrawer.tsx

```diff
@@ -80,7 +80,7 @@ const useStyles = makeStyles((theme: Theme) =>
       ...theme.mixins.toolbar,
     },
     content: {
-      padding: theme.spacing(3),
+      //padding: theme.spacing(3),
```

Comment:
Did you intend to completely remove the above line?

Reply:
yes, I removed it because, after changing the position to relative app bar box came with spacing so I had to remove this.